### ISSUE TITLE
Highlight and number matching instances in search results

### ISSUE THREAD

**Issue by jleaves** [sumatrapdfreader/sumatrapdf#1765]
First, thank you all the contributors to this great project. It's light and fast, yet powerful.

I would like to keep Sumatra as my only PDF reader. However, there is one feature that is missing: highlight and number all the matching instances in search results. I believe that a great number of users would like this feature as well.

Hopefully, it will

1) highlight all matching instances in PDF contents  
2) show the total number of matching instances, as well as which one you are currently at  (e.g. 5/117)
3) indicate the locations for all matching instances by placing a small horizontal line on the scrollbar on the right

Basically, similar to what Google Chrome does for PDF search results. Similar previous suggestions can be found [here](https://forum.sumatrapdfreader.org/t/highlight-and-number-search-results/252), but seem to have been left neglected in history.

At present, the search results only show the ONE matching instance that you are currently at. No numbers, no highlights for other instances, and no indication of the locations of all matching instances.

Any consideration for this? It'd be awesome if this feature could be implemented.

**Comment by GitHubRulesOK** [comment]
Apart from your sugestion to display and scrollbar indicate the current index position in overall count,  which was hinted by others in forum but seemingly not on git hub,  I have to count this as a duplicate of https://github.com/sumatrapdfreader/sumatrapdf/issues/154# however the issues will be consolidated.
